### PR TITLE
Update Cluster API Azure milestone plugin

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
https://github.com/kubernetes-sigs/cluster-api-provider-azure/milestones?with_issues=no 

/assign @nader-ziada @devigned

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -342,10 +342,9 @@ milestone_applier:
     release-1.17: v1.17
     release-1.16: v1.16
   kubernetes-sigs/cluster-api-provider-azure:
-    master: v0.4.11
-    release-0.4: v0.4
-    release-0.3: v0.3
-  kubernetes-sigs/cluster-api-provider-digitalocean:
+    master: v0.5.11
```
Comment (nader-ziada) [inline config/prow/plugins.yaml:345]: did you mean 0.5.0?

Comment (CecileRobertMichon) [reply config/prow/plugins.yaml:345]: oops yes I did

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**